src/app/pages/Profile/MyTrades/MyTrades.module.scss

```diff
@@ -0,0 +1,20 @@
+@import '/src/app/styles/base/settings';
+
+.myTrades {
+  margin: 2rem 0;
+  padding: 1rem;
+
+  &__header {
+    background-color: $black;
+    padding: 1rem;
+  }
+
+  &__content {
+    background-color: $background-promotion;
+    padding: 1rem;
+
+    &Row {
```

Comment:
Why isn't this class `&__row`? 
Style would be more consistent if the same way of naming classes is followed.

Reply:
It's because BEM convention. We are using the syntax: Module__Element--Modifier. As we are styling with scss modules our classnames should go as JS objects so we can not include -- in name. That's why we decided to use camel case.